src/exceptions/OpenJDK-Assembly-exception-2.0.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="OpenJDK-assembly-exception-1.0" name="OpenJDK Assembly exception 1.0">
+      <crossRefs>
+         <crossRef>http://openjdk.java.net/legal/assembly-exception.html
+         </crossRef>
+      </crossRefs>
+      <text>
+      <titleText>OpenJDK Assembly Exception</titleText>  
+      <p>
+         The OpenJDK source code made available by Oracle America, Inc.
+         (Oracle) at openjdk.java.net ("OpenJDK Code") is distributed
+         under the terms of the GNU General Public License
+         &lt;http://www.gnu.org/copyleft/gpl.html&gt; version 2 only
```

Comment:
Perhaps this link should be to (or at least allow) https://www.gnu.org/licenses/old-licenses/gpl-2.0.html?  Currently https://www.gnu.org/copyleft/gpl.html has the v3 text, and you have to click through “Old versions of the GNU GPL” and “GNU General Public License, version 2” to get from there to the v2 text.

Reply:
Also allowing `https` as well would be nice (like we do in the GPL licenses themselves since #450).

Reply:
> I like allowing http|https matches (let's encrypt! and all that), but that's broader than just this license. Let's open a separate issue for that.

With this sort of thing, I think the easiest approach is to gate-keep new submissions to hold the new standards while slowly chipping away at the existing instances.  Landing new code that violates our intended approach just makes the existing issues more trouble to fix ;).  This lets you solve the issues incrementally without having huge-change flag-days.

Reply:
I can see the case for that, but we don't want to hold up substantive additions to the license list while deciding. Also, if we want to systematically change this, there wouldn't appear to be much effort difference in (N licenses affected) vs. (N-1 licenses affected).

Set up in #633 to address systematically later.